### ISSUE TITLE
Error when running sample: ModuleNotFoundError: No module named 'google'

### ISSUE THREAD

**Issue by aaronpowell** [leikoilja/glocaltokens#257]
**Describe the bug**
I've followed the instructions in the README but when running it I receive the following error:

```
raceback (most recent call last):
  File "/workspaces/tmp-ha-google/get-token.py", line 1, in <module>
    from glocaltokens.client import GLocalAuthenticationTokens
  File "/home/vscode/.local/lib/python3.10/site-packages/glocaltokens/client.py", line 24, in <module>
    from .google.internal.home.foyer.v1_pb2 import GetHomeGraphRequest, GetHomeGraphResponse
  File "/home/vscode/.local/lib/python3.10/site-packages/glocaltokens/google/internal/home/foyer/v1_pb2.py", line 6, in <module>
    from google.protobuf import (
ModuleNotFoundError: No module named 'google'
```

This is in an isolated environment with python 3.10.4 installed (I ran a fresh Docker image with only Python 3 in it).

Full terminal logs:

```
vscode ➜ /workspaces/tmp-ha-google $ pip install glocaltokens
Defaulting to user installation because normal site-packages is not writeable
Collecting glocaltokens
  Downloading glocaltokens-0.6.4-py3-none-any.whl (37 kB)
Collecting grpcio<2.0.0,>=1.44.0
  Downloading grpcio-1.47.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (4.5 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 4.5/4.5 MB 8.5 MB/s eta 0:00:00
Collecting zeroconf<0.39.0,>=0.38.3
  Downloading zeroconf-0.38.7-py3-none-any.whl (106 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 106.6/106.6 KB 7.8 MB/s eta 0:00:00
Collecting requests<3.0.0,>=2.27.1
  Downloading requests-2.28.0-py3-none-any.whl (62 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 62.8/62.8 KB 4.2 MB/s eta 0:00:00
Collecting simplejson<4.0.0,>=3.17.2
  Downloading simplejson-3.17.6-cp310-cp310-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_12_x86_64.manylinux2010_x86_64.whl (137 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 137.1/137.1 KB 8.4 MB/s eta 0:00:00
Collecting gpsoauth<2.0.0,>=1.0.1
  Downloading gpsoauth-1.0.2-py3-none-any.whl (6.6 kB)
Collecting pycryptodomex>=3.0
  Downloading pycryptodomex-3.15.0-cp35-abi3-manylinux2010_x86_64.whl (2.3 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 2.3/2.3 MB 6.9 MB/s eta 0:00:00
Collecting six>=1.5.2
  Downloading six-1.16.0-py2.py3-none-any.whl (11 kB)
Collecting urllib3<1.27,>=1.21.1
  Downloading urllib3-1.26.9-py2.py3-none-any.whl (138 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 139.0/139.0 KB 8.8 MB/s eta 0:00:00
Collecting charset-normalizer~=2.0.0
  Downloading charset_normalizer-2.0.12-py3-none-any.whl (39 kB)
Collecting certifi>=2017.4.17
  Downloading certifi-2022.6.15-py3-none-any.whl (160 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 160.2/160.2 KB 9.8 MB/s eta 0:00:00
Collecting idna<4,>=2.5
  Downloading idna-3.3-py3-none-any.whl (61 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 61.2/61.2 KB 5.2 MB/s eta 0:00:00
Collecting ifaddr>=0.1.7
  Downloading ifaddr-0.2.0-py3-none-any.whl (12 kB)
Installing collected packages: ifaddr, zeroconf, urllib3, six, simplejson, pycryptodomex, idna, charset-normalizer, certifi, requests, grpcio, gpsoauth, glocaltokens
Successfully installed certifi-2022.6.15 charset-normalizer-2.0.12 glocaltokens-0.6.4 gpsoauth-1.0.2 grpcio-1.47.0 idna-3.3 ifaddr-0.2.0 pycryptodomex-3.15.0 requests-2.28.0 simplejson-3.17.6 six-1.16.0 urllib3-1.26.9 zeroconf-0.38.7
WARNING: You are using pip version 22.0.4; however, version 22.1.2 is available.
You should consider upgrading via the '/usr/local/bin/python -m pip install --upgrade pip' command.
vscode ➜ /workspaces/tmp-ha-google $ python3 get-token.py 
Traceback (most recent call last):
  File "/workspaces/tmp-ha-google/get-token.py", line 1, in <module>
    from glocaltokens.client import GLocalAuthenticationTokens
  File "/home/vscode/.local/lib/python3.10/site-packages/glocaltokens/client.py", line 24, in <module>
    from .google.internal.home.foyer.v1_pb2 import GetHomeGraphRequest, GetHomeGraphResponse
  File "/home/vscode/.local/lib/python3.10/site-packages/glocaltokens/google/internal/home/foyer/v1_pb2.py", line 6, in <module>
    from google.protobuf import (
ModuleNotFoundError: No module named 'google'
vscode ➜ /workspaces/tmp-ha-google $ 
```

**Comment by aaronpowell** [comment]
> Does it work if you install `google-api-python-client`?

Done, but looks like a protobuf problem now:

```
vscode ➜ /workspaces/tmp-ha-google $ python3 get-token.py 
Traceback (most recent call last):
  File "/workspaces/tmp-ha-google/get-token.py", line 1, in <module>
    from glocaltokens.client import GLocalAuthenticationTokens
  File "/home/vscode/.local/lib/python3.10/site-packages/glocaltokens/client.py", line 24, in <module>
    from .google.internal.home.foyer.v1_pb2 import GetHomeGraphRequest, GetHomeGraphResponse
  File "/home/vscode/.local/lib/python3.10/site-packages/glocaltokens/google/internal/home/foyer/v1_pb2.py", line 28, in <module>
    _GETASSISTANTROUTINESREQUEST = _descriptor.Descriptor(
  File "/home/vscode/.local/lib/python3.10/site-packages/google/protobuf/descriptor.py", line 313, in __new__
    _message.Message._CheckCalledFromGeneratedFile()
TypeError: Descriptors cannot not be created directly.
If this call came from a _pb2.py file, your generated code is out of date and must be regenerated with protoc >= 3.19.0.
If you cannot immediately regenerate your protos, some other possible workarounds are:
 1. Downgrade the protobuf package to 3.20.x or lower.
 2. Set PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python (but this will use pure-Python parsing and will be much slower).

More information: https://developers.google.com/protocol-buffers/docs/news/2022-05-06#python-updates
```

**Comment by aaronpowell** [comment]
Ok, downgraded `protobuf` with `pip install protobuf==3.19.4`

Is there a way to pin the dependency so it doesn't install too newer version?

**Comment by KapJI** [comment]
@aaronpowell I released version 0.6.5 with a fix. Can you confirm that it works without installing `google-api-python-client` and downgrading `protobuf`?
